### ISSUE TITLE
empty email in service account

### ISSUE THREAD

**Issue by pbavinck** [kbhattac/coolretailer#3]
Starting deploy.sh

Terminal shows:
.  Starting the setup process in project coolretailer-2... 
.  Creating a service account coolretailer-app-1553854075... 
Created service account [coolretailer-app-1553854075].
Service Account email is empty. Exiting.

Running the deploy.sh again results in error that service accounts already exists, but does continue afterwards.

Perhaps insert short delay after service account is created?

**Comment by kbhattac** [comment]
Did you set the project first: `gcloud config set project {project-id}`

**Comment by kbhattac** [comment]
added logic to wait for 10s. Checking now.

**Comment by pbavinck** [comment]
10s was not enough. Failed. Restarting script failed at creating service account as at that point the SA existed. Luckily the script continues after that.

My suggestion would be to use somthing inspired by:
```
while [ ! -f /tmp/list.txt ]
do
  sleep 2
done
ls -l /tmp/list.txt
```

**Comment by kbhattac** [comment]
Yes; that's why this issues is still open. Will fix it. Pull is request is of course more than welcome ;)
